File: Service_eHospital_DongNai_A/Implement/me_MAC_MappingService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class me_MAC_MappingService : BaseService<me_MAC_Mapping, Ime_MAC_MappingRepository>
    , Ime_MAC_MappingService
    {
    private readonly Ime_MAC_MappingRepository _me_MAC_MappingRepository;
    public me_MAC_MappingService(Ime_MAC_MappingRepository me_MAC_MappingRepository) : base(me_MAC_MappingRepository)
    {
    _me_MAC_MappingRepository = me_MAC_MappingRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A/me_MAC_Mapping.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { me_MAC_Mapping } from './me_MAC_Mapping.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class me_MAC_MappingService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Mapping_id', 'MAC_id', 'Cua_id', 'KhuVuc_id', 'TamNgung', 'NgayMapping', 'SoDaCap', 'ThoiGianCap', 'SoDaGoi', 'ThoiGianGoi'];

    List: me_MAC_Mapping[] | undefined;
    ListFilter: me_MAC_Mapping[] | undefined;
    FormData!: me_MAC_Mapping;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "me_MAC_Mapping";
    }
}

